src/test/ij.c

```diff
@@ -3404,115 +3404,33 @@ main( hypre_int argc,
       HYPRE_BoomerAMGSetADropTol(amg_solver, A_drop_tol);
       HYPRE_BoomerAMGSetADropType(amg_solver, A_drop_type);
       /* BM Aug 25, 2006 */
-      HYPRE_BoomerAMGSetCGCIts(amg_solver, cgcits);
```

Comment:
@oseikuffuor1 why should we make these deletions?

Reply:
Oops thanks for catching that. I edited it for testing forgot to checkout a clean version. I will correct that.

Reply:
@liruipeng corrected the update. Thanks.